internal/Configuration.go

```diff
@@ -0,0 +1,13 @@
+package internal
+
+import "github.com/caarlos0/env"
+
+type Configuration struct {
+	CommitStatsTimeoutInSec int `env:"COMMIT_STATS_TIMEOUT_IN_SEC" envDefault:"2"`
+}
+
+func ParseConfiguration() (*Configuration, error) {
+	cfg := &Configuration{}
```

Comment:
We can make this more intuitive like CommitTimeoutConfiguration

Reply:
This configuration is generic. (Think it as adding more configurations in future in this. )